website/content/gateway/getting_started/response_format/response_format.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Response_format"
+date: 2022-04-12T13:49:00+02:00
+anchor: "response-format"
+weight: 25
+---
+## Response format
+All responses will be delivered in JSON format (see [JSON-HAL](https://tools.ietf.org/html/draft-kelly-json-hal-08)).
+
+```
+Content-Type: application/vnd.clearhaus-gateway.hal+json; version=0.10.0; charset=utf-8
+```
+where the version follows [Semantic Versioning](https://semver.org/).
+We use HTTP response codes to indicate API response status:
```

Comment:
```suggestion

We use HTTP response codes to indicate API response status:
```

Newline is required or the two sentences will be on the same line in the rendered output. Note that they are not on the same line in https://docs.gateway.clearhaus.com/#response-format.